docs/topics/forms/formsets.txt

```diff
@@ -448,6 +448,21 @@ happen when the user changes these values::
     {'pub_date': datetime.date(2008, 5, 11), 'ORDER': 1, 'title': 'Article #2'}
     {'pub_date': datetime.date(2008, 5, 10), 'ORDER': 2, 'title': 'Article #1'}
 
+You can change the order field widget for :class:`~django.forms.ModelForm`
```

Comment:
We should have proper sections here for both `ordering_widget` and `get_ordering_widget()`

e.g:

```
.. attribute:: BaseFormSet.ordering_widget
```

(You'd override `get_ordering_widget()` to provide an instance...)